packages/gasket-typescript-tests/test/plugin-log.spec.ts

```diff
@@ -30,4 +31,41 @@ describe('@gasket/plugin-log', () => {
       ];
     }
   });
+
+  it('adds a logger property to Gasket', () => {
+    const logger: Log = {
+      alert(...args: any): void {},
+      crit(...args: any): void {},
+      debug(...args: any): void {},
+      emerg(...args: any): void {},
+      error(...args: any): void {},
+      info(...args: any): void {},
+      warning(...args): void {},
+      notice(...args): void {},
+      log(...args: any): void {},
+
+      close(): Promise<any> {
+        return Promise.resolve(undefined);
+      },
+    };
+  });
+
+  it('logger instance allows custom level method', () => {
+    const logger2: Log = {
+      alert(...args: any): void {},
+      crit(...args: any): void {},
+      debug(...args: any): void {},
+      emerg(...args: any): void {},
+      error(...args: any): void {},
+      info(...args: any): void {},
+      warning(...args): void {},
+      notice(...args): void {},
+      log(...args: any): void {},
+
+      // @ts-ignore-error
```

Comment:
Is the TS error because it expects a `Promise` return type? You could add `async` in front of close if that's the reason.